app/src/lib/stores/app-store.ts

```diff
@@ -5266,9 +5266,11 @@ export class AppStore extends TypedBaseStore<IAppState> {
   ): Promise<void> {
     try {
       if (moveToTrash) {
-        const deleted = shell.moveItemToTrash(repository.path)
+        try {
+          await shell.moveItemToTrash(repository.path)
+        } catch (error) {
+          log.error(error)
```

Comment:
Could we add a little message here to differentiate errors in the log?

```suggestion
          log.error('Failed moving repository to trash`, error)
```